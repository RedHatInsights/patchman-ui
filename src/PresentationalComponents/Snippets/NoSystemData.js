import { Bullseye, Button, EmptyState, EmptyStateBody, EmptyStateIcon, Title } from '@patternfly/react-core';
import { WrenchIcon } from '@patternfly/react-icons';
import React from 'react';

export const NoSystemData = () => (
    <Bullseye>
        <EmptyState style={{ paddingTop: 40 }}>
            <EmptyStateIcon icon={WrenchIcon} />
            <Title headingLevel="h5" size="lg">
                Patch is not yet configured
            </Title>
            <EmptyStateBody>
                Activate the Insights client
            </EmptyStateBody>
            <Button
                variant="primary"
                component="a"
                href="http://access.redhat.com/products/cloud_management_services_for_rhel#getstarted"
            >
                Learn about the Insights client
            </Button>
        </EmptyState>
    </Bullseye>
);
