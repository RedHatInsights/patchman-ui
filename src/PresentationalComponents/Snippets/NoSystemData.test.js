import { NoSystemData } from './NoSystemData';
import toJson from 'enzyme-to-json';

const wrapper = shallow(<NoSystemData />);

describe('NoSystemData', () => {
    it('Should match the snapshot', () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
