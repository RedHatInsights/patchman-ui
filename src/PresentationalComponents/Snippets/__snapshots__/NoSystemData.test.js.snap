// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoSystemData Should match the snapshot 1`] = `
<Bullseye>
  <EmptyState
    style={
      Object {
        "paddingTop": 40,
      }
    }
  >
    <EmptyStateIcon
      icon={[Function]}
    />
    <Title
      headingLevel="h5"
      size="lg"
    >
      Patch is not yet configured
    </Title>
    <EmptyStateBody>
      Activate the Insights client
    </EmptyStateBody>
    <Button
      component="a"
      href="http://access.redhat.com/products/cloud_management_services_for_rhel#getstarted"
      variant="primary"
    >
      Learn about the Insights client
    </Button>
  </EmptyState>
</Bullseye>
`;
